site/hxapi/transfers/bkg/index.md

```diff
@@ -33,48 +33,50 @@ The parameters _must_ be sent in the body of the request, as `x-www-form-urlenco
+| LastName           | String    | [A-Z]            | Y          | The lead passenger's last name.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| Email              | String    | [A-Z] 50 chars   | Y          | The lead passenger's email address.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| DayPhone           | Integer   | [0-9] 20 chars   | Y          | The lead passenger's contact telephone number.                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| Price              | Float     | [0-9]            | Y          | The total price of the Transfer/s (This will be the value of the `TotalPrice` property returned in the availability response).                                                                                                                                                                                                                                                                                                                                                                                           |
+| PriceCheckFlag     | String    | Y                | Y          | Indicates a price check will be carried out prior to booking. <br>If the price is lower in the system the booking will be made as the customer will benefit from a saving.  If the price has changed and is higher than the price stated on availability, an error will return to explain that the price has increased, the booking can still be made but the customer must confirm that they will pay the higher amount.  You should then resubmit the request changing the PriceCheckPrice with the new price.         |
```

Comment:
So it's mandatory, and the only possible value is 'Y'? That doesn't make much sense.

(And if it can be unspecified or set to N/false, then PriceCheckPrice below shouldn't be mandatory.)

Reply:
think Gio took this from the docs for hotel and parking, they specify the same here